### ISSUE TITLE
Response from PLOS Computation Biology

### ISSUE THREAD

**Issue by Benjamin-Lee** [Benjamin-Lee/deep-rules#99]
>
>Thanks for your Presubmission Inquiry to PLOS Computational Biology regarding your manuscript 'Ten Simple Rules for Deep Learning in Biology'. The abstract appears within the scope of the journal and we would welcome the opportunity to consider the full paper.
>
>Although we cannot make any guarantees as to the outcome, especially prior to evaluating the full manuscript, please feel encouraged to submit the complete manuscript (at https://www.editorialmanager.com/pcompbiol) and we will assess your paper for suitability and then, if warranted, send it for external peer review.
>
>That said, we will not consider this under the rubric of "ten simple rules" (which is
suppose to have a career focus), but rather under the new “quick tips” section of the PLOS Computational Biology Education Collection. This new section (e.g. see "Ten Quick Tips for Using the Gene Ontology" https://goo.gl/9PwgWp ),  is geared towards databases, resources and tools and is indeed suppose to have an education slant to them, like yours does. An advantage of the Quick Tips is that you are are not restricted to the number “10”. If it make sense to have 8 or 9, or 13, please feel free to do so. The title for your submission should be something like “Eleven Quick Tips for Deep Learning in Biology”, and like any other article should reference similar work from PLOS and other publishers. It could have one or two figures and/or table, and you should limit it to 2,500-3,000 words. 
>
>When you do submit, please follow the Author Guidelines (http://www.ploscompbiol.org/static/guidelines.action) and include mention of this Presubmission Inquiry and the manuscript number (PCOMPBIOL-D-18-02064) in the Previous Interactions section of the research article submission form - this will help us find this response letter. Thank you.
>
>Yours sincerely,
>
>@bffo
>
>BF Francis Ouellette
>Education Editor
>PLOS Computational Biology

Do we want to change the title to "quick tips" or send an inquiry to another journal?

**Comment by cgreene** [comment]
Quick tips seems ideal. Particularly as our number can differ slightly (though I like having a fixed number to focus on during development). 🎉

**Comment by rasbt** [comment]
> Do we want to change the title to "quick tips" or send an inquiry to another journal?

Either way would be fine with me. I must say though that "10 quick tips ..." sounds like one of these buzzword blog articles or so :P 

On the other hand, the things we are writing about, these really are tips, not "rules"

**Comment by sgfin** [comment]
I agree that this is a fine option.  I also agree that the numbered title sounds a bit like a buzzfeed article (applies to "rules" as well as "tips", though, IMO).  It would be nice if we could use a more traditional title.

**Comment by jmschrei** [comment]
This sounds good. I don't like being forced into a fixed number of rules, regardless, so that you can focus  on succinctly writing the really important stuff. 🎉

**Comment by agitter** [comment]
> Quick tips seems ideal.

I definitely agree.  This seems like a very positive outcome, and I vote to stay at PLOS Computational Biology with the new format.

I see five current [quick tips articles](https://collections.plos.org/compbiol-education):
- [Ten quick tips for teaching programming](http://dx.plos.org/10.1371/journal.pcbi.1006023)
- [Eleven quick tips for running an interdisciplinary short course for new graduate students](http://dx.plos.org/10.1371/journal.pcbi.1006039)
- [Eleven quick tips for architecting biomedical informatics workflows with cloud computing](http://dx.plos.org/10.1371/journal.pcbi.1005994)
- [Quick tips for creating effective and impactful biological pathways using the Systems Biology Graphical Notation](http://dx.plos.org/10.1371/journal.pcbi.1005740)
- [Ten Quick Tips for Using the Gene Ontology](http://dx.plos.org/10.1371/journal.pcbi.1003343)

If we agree to go this route, we can review those to get a feel for the style.  It looks fairly similar to Ten Simple Rules at first glance.

**Comment by SiminaB** [comment]
Nice! I would vote for less than or equal to 10 tips. :-) "Eleven quick tips" just sounds like maybe they're not so quick, after all.

**Comment by AlexanderTitus** [comment]
> Nice! I would vote for less than or equal to 10 tips. :-) "Eleven quick tips" just sounds like maybe they're not so quick, after all.

Agreed!

**Comment by rasbt** [comment]
> Nice! I would vote for less than or equal to 10 tips. :-) "Eleven quick tips" just sounds like maybe they're not so quick, after all.

Yes, totally agree with that. Since we have that 2500-3000 words limit, fewer "tips" would help explaining these more clearly. But looking over the rules right now, I think the 10 we have are pretty good. I (almost) wouldn't know which ones to remove ;). 

- Maybe Rule 3 (Understand the complexities of training deep neural networks) & 6 (Tune your hyperparameters extensively and systematically) can be merged
- Also, Rule 4 (Know your data and your question) & 5 (Choose an appropriate neural network architecture and data representation (#29)) could be merged.

If we are generally okay with reducing the number of rules, I can open a PR (and continue the discussion there).

> I think PLOS CB is an ideal audience for this work.

Yes, I saw that one too today, when I was searching for the author guidelines. Is it a recent, newer sub-"magazine"? Because -- haven't heard of the Computational Biology one before. On the other hand, the Biology one might not be too bad as a superset, to also capture the computational-technique-practitioners-(vs developers)-oriented audience who might be interested in using DL (CB always sounds like a more advanced audience who might find our article to trivial).

**Comment by agitter** [comment]
> Is it a recent, newer sub-"magazine"?

PLOS Computational Biology has been around since ~2005 so it is only about 2 years newer than PLOS Biology.  It is firmly established as a top bioinformatics journal in my opinion.

The PLOS Biology audience may benefit more from these "quick tips", but PLOS Computational Biology editors are likely more understanding of why the article is needed in the first place.  PLOS Biology also has a more [limited range of education articles](https://journals.plos.org/plosbiology/s/other-article-types#loc-education).

**Comment by AlexanderTitus** [comment]
> PLOS Computational Biology has been around since ~2005 so it is only about 2 years newer than PLOS Biology. It is firmly established as a top bioinformatics journal in my opinion.

I agree. PLOS CB along with Bioinformatics are arguably the top two bioinformatics journals and PLOS CB  is a strong choice.

**Comment by pstew** [comment]
This is great! Big thumbs up for sticking with PLOS Computational Biology as a "tips" article. PCB has a very good reputation.

There was some commentary in the issues about the similarity of rules and if some needed to be combined. Since we're not strictly married to 10, maybe we can see where the writing takes us and if it makes sense to combine down the road?

**Comment by rasbt** [comment]
I am not that familiar with PLoS but that sounds all great, an even better fit than PLoS Bio then!

**Comment by michaelmhoffman** [comment]
@Benjamin-Lee What's next? I would suggest we start by re-evaluating the list of top-level rules since we don't have 9. Might be good to have a due date for any comments on that.

**Comment by Benjamin-Lee** [comment]
Since the consensus is to stick with PLOS CB and change it to "tips" from "rules", we'll want to:

1. Change wording on the repo about rules into tips via a PR
2. Reevaluate the top level list
3. Once reevaluated, update the repo to the reflect the consensus via a PR

On a personal note, I have my last final exam on the 18th and am then flying out the next day to be a lab rat in a [clinical trial](https://clinicaltrials.gov/ct2/show/NCT03347708), so I am going to be sporadic for about the next week. I'll make issues about those tasks and if anyone wants to assign themselves, they can feel free.

**Comment by chevrm** [comment]
How much of a rule is having "quick tips" in the title? "Guidelines" or similar could distance us from the buzzfeed feel. 

Either way is great. PLoS CB is an ideal fit IMO.

**Comment by agitter** [comment]
> How much of a rule is having "quick tips" in the title? "Guidelines" or similar could distance us from the buzzfeed feel.

We could always ask the editor, but it looks like a requirement for the article format.  I updated my comment above (https://github.com/Benjamin-Lee/deep-rules/issues/99#issuecomment-446777915) with two new quick tips articles.  All of the published examples include "quick tips", though one of them does not include the number of tips.  I also could have missed examples that don't use "quick tips" in the title.

**Comment by AlexanderTitus** [comment]
This has been agreed upon and is in process.
